### ISSUE TITLE
Language Switch

### ISSUE THREAD

**Issue by ollimaz** [nabeelio/phpvms#740]
I would like to give a user the option to switch between different languages. 

It could be done with some kind of dropdown menue in the footer. The User can click on different languages that have been added to the resources/lang/ folder. The current language could be saved in a session cookie.

**Comment by nabeelio** [comment]
Hello, I will add this into the roadmap, I have to see if Laravel will support it being changed during runtime (since it's set in the config file right now). Thanks!

**Comment by ollimaz** [comment]
You are right! It doesn't but this https://github.com/akaunting/language might help?!

**Comment by nabeelio** [comment]
It could, nice find!

**Comment by ollimaz** [comment]
When are you likely to implement this feature? Or do you need help? If so, the question is, how to handle custom created pages...

Furthermore, the Issue #777 and this issue are linked in someway

**Comment by nabeelio** [comment]
Yes it's related to that; I don't know when, probably not soon

**Comment by arv187** [comment]
https://blog.quickadminpanel.com/10-best-laravel-packages-for-multi-language-translations/
If this is useful to collaborators help you to translate

**Comment by ollimaz** [comment]
#1020

**Comment by arthurpar06** [comment]
I think #1500 closes this issue
